src/highlight/highlight.ts

```diff
@@ -9,30 +10,17 @@ import { bem } from '../utils';
 export class NgAisHighlight {
   @Input() attribute: string;
   @Input() hit: { _highlightResult?: {}; label?: string; highlighted?: string };
-  @Input() tagName: string = 'em';
+  @Input() tagName: string = 'mark';
 
   cx = bem('Highlight');
 
   get content() {
-    if (this.attribute === 'highlighted') {
-      return this.hit.highlighted
-        ? this.replaceWithTagName(this.hit.highlighted)
-        : this.hit.label;
-    }
-
     if (this.hit.hasOwnProperty('_highlightResult')) {
```

Comment:
We should now solely rely on the [`highlight` helper](https://github.com/algolia/instantsearch.js/blob/af98a46543509d5101586242b4be14a64ed2762d/src/helpers/highlight.ts). We don't need other checks/fallbacks because it handles all cases for us.

Reply:
yes, it can safely be removed here.